pvlib/inverter.py

```diff
@@ -91,29 +124,80 @@ def sandia(v_dc, p_dc, inverter):
     '''
 
     Paco = inverter['Paco']
-    Pdco = inverter['Pdco']
-    Vdco = inverter['Vdco']
-    Pso = inverter['Pso']
-    C0 = inverter['C0']
-    C1 = inverter['C1']
-    C2 = inverter['C2']
-    C3 = inverter['C3']
     Pnt = inverter['Pnt']
+    Pso = inverter['Pso']
 
-    A = Pdco * (1 + C1 * (v_dc - Vdco))
-    B = Pso * (1 + C2 * (v_dc - Vdco))
-    C = C0 * (1 + C3 * (v_dc - Vdco))
-
-    power_ac = (Paco / (A - B) - C * (A - B)) * (p_dc - B) + C * (p_dc - B)**2
-    power_ac = np.minimum(Paco, power_ac)
-    power_ac = np.where(p_dc < Pso, -1.0 * abs(Pnt), power_ac)
+    power_ac = _sandia_eff(v_dc, p_dc, inverter)
+    power_ac = _sandia_limits(power_ac, p_dc, Paco, Pnt, Pso)
 
     if isinstance(p_dc, pd.Series):
         power_ac = pd.Series(power_ac, index=p_dc.index)
 
     return power_ac
 
 
+def sandia_multi(v_dc, p_dc, inverter):
+    r'''
+    Convert DC power and voltage to AC power for an inverter with multiple
+    MPPT inputs.
+
+    Uses Sandia's Grid-Connected PV Inverter model [1]_.
+
+    Parameters
+    ----------
+    v_dc : numeric or tuple of numeric
+        DC voltage on each MPPT input of the inverter. [V]
+
+    p_dc : numeric or tuple of numeric
+        DC power on each MPPT input of the inverter. [W]
```

Comment:
what's the use case for allowing numeric? Can we always require tuple/list of numeric?

Reply:
to have this function behave appropriately when only one MPPT input is being used. In that case I'm not opposed to requiring the input to be `list or tuple of numeric` of length 1.

Reply:
+1 to requiring an Iterable of length 1

Reply:
Do we want to test for and enforce `tuple` or `list` type? Duck-typing risks returning garbage results if the input type is array.

Reply:
I'd prefer a garbage in, garbage out approach in this situation because it feels like we're still figuring out this function.